README.md

```diff
@@ -53,7 +53,7 @@ Check out the [online documentation](https://openfl.readthedocs.io/en/latest/ind
 - Ubuntu Linux 18.04+
 - Python 3.7+ (recommended to use with [Virtualenv](https://virtualenv.pypa.io/en/latest/)).
 
-OpenFL supports training with TensorFlow 2+ or PyTorch 1.3+ which should be installed separately. User can extend the list of supported Deep Learning frameworks if needed.
+OpenFL supports training with TensorFlow 2+ or PyTorch 2+ which should be installed separately. Users can extend the list of supported Machine Learning frameworks if needed.
 
 ## Project Overview
 ### What is Federated Learning
```

Comment:
The `### Supported Aggregation Algorithms` table could be reworked. I think the aggregation algorithms are generally framework agnostic since the weights have been reduced to layer-based np arrays by the time it gets to this stage. Also, the docs all point to the same page. maybe we just keep it at `| algorithm name | paper |` ?

Reply:
Thank for pointing this out, @kta-intel. For now I propose a light-weight approach where I just remove the final column which is indeed redundant. In follow-up PRs we can also update the core content of this paragraph.